### PR TITLE
New package: dustinblackman.oatmeal 0.12.3

### PR DESCRIPTION
**Pull request by dustinblackman**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132355)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/d/dustinblackman/oatmeal/0.12.3/dustinblackman.oatmeal.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - Architecture: arm64
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: oatmeal.exe
+        PortableCommandAlias: oatmeal
+    InstallerUrl: https://github.com/dustinblackman/oatmeal/releases/download/v0.12.3/oatmeal_0.12.3_windows_arm64.zip
+    InstallerSha256: df912c32871470ed1d32adbd79b9310dfb3b4dd6d2dd1385b67db37a524df899
+    UpgradeBehavior: uninstallPrevious
+  - Architecture: x64
+    NestedInstallerType: portable
```
Comment (stephengillie) [inline manifests/d/dustinblackman/oatmeal/0.12.3/dustinblackman.oatmeal.installer.yaml:18]:
```suggestion
    Dependencies:
      PackageDependencies:
        - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    NestedInstallerType: portable
```

Comment (stephengillie) [reply manifests/d/dustinblackman/oatmeal/0.12.3/dustinblackman.oatmeal.installer.yaml:18]: Unfortunately, we don't currently have an `arm` equivalent to this dependency.

**Comment by stephengillie** [comment]
With the dependency installed, the application gives this message:
image

Should "Ollama" be a dependency? We don't currently have a manifest for this package either.

**Comment by dustinblackman** [comment]
Hi there!

Nope, Oatmeal consumes the Ollama API which just isn't running on the test machine. Expected behaviour.

What changes can I make to fill the initial error?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**